source/geod24/concurrency.d

```diff
@@ -107,6 +107,7 @@ private
     {
         MsgType type;
         Variant data;
+        MonoTime create_time;
```

Comment:
What is this for ?

Reply:
It is necessary to measure the wait time.